k8s-install/scripts/install-cni.sh

```diff
@@ -6,7 +6,7 @@
 # - Expects the desired CNI config in the CNI_NETWORK_CONFIG env variable.
 
 # Ensure all variables are defined.
```

Comment:
comment should be updated to say "and that the script fails when an error is hit"